### PR TITLE
Table restructure for our needs

### PR DESCRIPTION
**Pull request by arnoldyahad**

### CODE REVIEW

File: robusta_krr/formatters/table.py [hunk 1]
```diff
@@ -81,19 +81,16 @@ def table(result: Result) -> Table:
...
+    table.add_column("Pods", style="cyan", no_wrap=True)
+    table.add_column("Container", style="cyan", no_wrap=True)
     for resource in ResourceType:
         table.add_column(f"{resource.name} Diff")
         table.add_column(f"{resource.name} Requests")
         table.add_column(f"{resource.name} Limits")
 
     for _, group in itertools.groupby(
-        enumerate(result.scans), key=lambda x: (x[1].object.cluster, x[1].object.namespace, x[1].object.name)
+        enumerate(result.scans), key=lambda x: (x[1].object.cluster, x[1].object.namespace, x[1].object.pods, x[1].object.current_pods_count)
```
Comment (Avi-Robusta) [inline robusta_krr/formatters/table.py:93]: so i think pods is an array of Objects so you would want to use len(x[1].object.pods) if you are doing it by pods amount

File: robusta_krr/formatters/table.py [hunk 2]
```diff
@@ -81,19 +81,16 @@ def table(result: Result) -> Table:
...
-    table.add_column("Old Pods", style="cyan")
-    table.add_column("Type", style="cyan")
-    table.add_column("Container", style="cyan")
+    table.add_column("Name", style="cyan", no_wrap=True)
+    table.add_column("Pods", style="cyan", no_wrap=True)
+    table.add_column("Container", style="cyan", no_wrap=True)
     for resource in ResourceType:
         table.add_column(f"{resource.name} Diff")
         table.add_column(f"{resource.name} Requests")
         table.add_column(f"{resource.name} Limits")
```
Comment (Avi-Robusta) [inline robusta_krr/formatters/table.py:91]: you would want to add a sort here before the grouping like this for example and have the grouping as it was 
`result.scans.sort( key=lambda x: (x.object.current_pods_count, x.object.cluster, x.object.namespace, x.object.name), reverse=True)`

Comment (Avi-Robusta) [reply robusta_krr/formatters/table.py:91]: I kept 
x.object.cluster, x.object.namespace, x.object.name
so the grouping stays the same

Comment (arnoldyahad) [reply robusta_krr/formatters/table.py:91]: Used the `result.scans.sort` and it works! thank you

**Pull request merged**